src/query/sql/src/executor/physical_plan.rs

```diff
@@ -660,6 +665,34 @@ impl UnionAll {
     }
 }
 
+#[derive(Clone, Debug, serde::Serialize, serde::Deserialize)]
+pub struct DistributedCopyIntoTableFromText {
```

Comment:
What's the `DistributedCopyIntoTableFromText ` FromText mean?

Reply:
by the way, the DistributedCopyIntoTableFromText's semantics is changed, I will use from stage? DistributedCopyIntoTableFromStage? what about it?

Reply:
@youngsofun may have some suggestion here, COPY have many data sources Stage is one of them.

Reply:
I think think from stage is well, because this pr just support copy into table from @stage, like copy into table from query or other things haven't been supported. (they will be provided in next pr)

Reply:
I suggest `DistributedCopyIntoTable`,  `CopyIntoTable` can only be `from stage`

Reply:
ok